java-client/session-dagger/src/test/java/io/deephaven/client/impl/TableServiceAsyncTest.java

```diff
@@ -79,22 +85,30 @@ private static TableHandle get(TableHandleFuture future)
         return future.getOrCancel(GETTIME);
     }
 
-    private static void checkSucceeded(TableHandle x) {
+    private void checkSucceeded(TableHandle x, int chainLength) {
         assertThat(x.isSuccessful()).isTrue();
+        try (final SafeCloseable ignored = getExecutionContext().open()) {
+            final Table result = getSession(session.getCurrentToken()).<Table>getExport(x.exportId().id()).get();
```

Comment:
I think we should capture the server-side session in `DeephavenSessionTestBase` field; you should have access to `SessionImpl` w/ return call of `io.deephaven.client.SessionSubcomponent#newSession`